tests/upgrade.bats

```diff
@@ -1,56 +1,59 @@
 #!/usr/bin/env bats
 
-source $BATS_TEST_DIRNAME/common.bash
+setup() {
+	load common.bash
+	wait_pods
+}
 
 @test "[CRD upgrade] Install old Kubewarden version" {
-	helm install --wait --kube-context $CLUSTER_CONTEXT \
```

Comment:
:100:  I look forward to just set the context at the beginning, or inherit it from outside of the testsuite, so the testsuite can be run against whatever context is set.